roles/xsnippet_api/templates/systemd.service.j2

```diff
@@ -2,6 +2,7 @@
 Description = XSnippet API
 After = network.target network-online.target
 Wants = network-online.target
+StartLimitIntervalSec = 0
```

Comment:
Are you sure about zero? Please correct me if I'm wrong, but your intention here is to disable start limiting intervals. If so, according to the docs you have to use the word `infinity` here instead. The value of `0` can be used to disable this globally for all services via `DefaultStartLimitIntervalSec` passed to systemd manager, but this is a different story.

https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#StartLimitIntervalSec=interval

Reply:
I find the wording in the doc very confusing, but, the way I interpret it is that systemd will allow up to `StartLimitBurst` restarts over the `StartLimitIntervalSec` seconds, and `0` is used to disable rate-limiting altogether while `infinity` enforces the rate-limit over the infinite period of time.

Reply:
> Defaults to `DefaultStartLimitIntervalSec=` in manager configuration file, and may be set to `0` to disable any kind of rate limiting.

Ah, okay, most likely I misread it. I think you're right now, and _«my be set to 0»_ is applied to the `StartLimitIntervalSec=`, not `DefaultStartLimitIntervalSec=`.

Reply:
Yeah. I think they could have described it better.